### ISSUE TITLE
Add certain functions to the dropdown menu

### ISSUE THREAD

**Issue by mstgrv** [xor-gate/syncthing-macosx#42]
It would be cool if certain Syncthing functions could be accessible from the syncthing-osx dropdown menu - basic things from the front page of the GUI, such as 

Pause All
Rescan All
Add Folder
Recent Changes
Add New Device
Settings
Restart

**Comment by xor-gate** [comment]
I have been thinking some time ago to add indeed some handy actions in the dropdown. However I would not like to reimplement dialogs which are available in the webinterface.

**Comment by mstgrv** [comment]
Yeah putting too many things in the dropdown would be bad, but the 2 things that would absolutely be handy to have would be "Pause All" and "Refresh All"

**Comment by xor-gate** [comment]
Yes I agree on those items.

**Comment by virusman** [comment]
Do you mean "Pause All"/"Resume All" and "Rescan All"?

**Comment by xor-gate** [comment]
Exactly, especially the pause/resume all item would be handy (e.g if tethering on a mobile data connection)

**Comment by mstgrv** [comment]
Also handy for when you've made a change you'd like to "commit" immediately, rather than waiting for the rescan interval (or filesystem watcher) to catch it

**Comment by xor-gate** [comment]
Filesystem watching is almost instant, not supported everywhere.

**Comment by virusman** [comment]
Pause all folders or pause all devices?

**Comment by xor-gate** [comment]
Good question, I think pause all devices.

**Comment by virusman** [comment]
See #43

**Comment by mstgrv** [comment]
I would say Pause All Folders is something worth adding as well - this is about saving energy/cpu cycles on the machine, rather than pausing syncing to peers (whilst keeping folders scanning)

**Comment by xor-gate** [comment]
@mstgrv yes I agree on this item.

**Comment by xor-gate** [comment]
Hi guys with latest release the Pause/Resume all devices and Rescan all is alive! The Pause All folders is still missing.

**Comment by mstgrv** [comment]
Thanks very much! Would be cool to see Pause All Folders introduced at some point, but it's great to have Rescan All and Pause All Devices in the menu now!

**Comment by vincentardern** [comment]
should this issue title be modified to represent the current status, or closed and a new issue opened for the items that are still left? would show the project is making progress, but could also be a waste of time.

**Comment by entonio** [comment]
For me, `Recent Changes` would be lovely - it really shows that things are happening - but would it be a submenu of the folder, or could we have a global one? My vote would be for the global one, so as to be as quick as possible to have something visible. Simple file name and date would be enough for me.

The Add functions look to me like the kind of thing that id out of scope on a 'quick access' widget like this.

**Comment by xor-gate** [comment]
I think the menu is already full of the necessary elements except for the proposed and useful Recent Changes so I changed the title of the issue.
